### ISSUE TITLE
Fulcro Inspect tab completely blank in latest Chrome

### ISSUE THREAD

**Issue by avocade** [fulcrologic/fulcro-inspect#94]
Had this problem for some time now (a year+?), where Fulcro Inspect just doesn't work at all in Chrome. I've tried everything, including uninstalling everything related to Google from my macOS (latest Catalina), and then setting up Chrome from scratch. Still shows a blank page.

**Comment by avocade** [comment]
PS. The extension icon for "Fulcro Inspect" lights up when I'm on eg the [Fulcro Book](http://book.fulcrologic.com) page, and says it's found a fulcro app. So everything seems to work up until the point where I view the inspector.

**Comment by msladecek** [comment]
I'm experiencing the same issue with chromium snap package
```
Chromium latest/stable 85.0.4183.102 2020-09-09
```
and also with the debian package
```
Chromium 83.0.4103.116 built on Debian 10.4, running on Debian bullseye/sid
```
on
```
KDE Plasma 5.18.5
Ubuntu 20.04.1 LTS

```

Testing on https://book.fulcrologic.com

Extension icon showed "Has access to this site" on hover and "Fulcro app detected" on click.

Fulcro Inspect tab in devtools was completely blank.

Running devtools again on the Fulcro inspect tab showed this message in console:
```
main.js:4106 Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at chrome-extension://meeijplnfjcihnhkpanepcaffklobaal/js/devtool/main.js:4106:162
    at chrome-extension://meeijplnfjcihnhkpanepcaffklobaal/js/devtool/main.js:5600:4
```

Googling that message led me to this:
https://stackoverflow.com/questions/30481516/iframe-in-chrome-error-failed-to-read-localstorage-from-window-access-deni

### TLDR Solution that worked for me:
1. Go to `chrome://settings/content/cookies`
2. uncheck `Block third-party cookies`
3. restart browser

**Comment by awkay** [comment]
You could follow the above (run devtools on Inspect) and look for errors. I can't help otherwise. It works for me.

**Comment by chrismurrph** [comment]
I also had problems with Fulcro Inspect and about a year ago went with: https://github.com/fulcrologic/fulcro-inspect#using-the-fulcro-3x-electron-app. The electron app has worked out well for me. I'm on a kind of slow Linux machine.

**Comment by avocade** [comment]
> ### TLDR Solution that worked for me:
> 1. Go to `chrome://settings/content/cookies`
> 2. uncheck `Block third-party cookies`
> 3. restart browser

Very interesting, thanks. It seems to work fine now for me (finally!) on a new Chrome Canary installation, but when I checked "Block third-party cookies" in Settings it stopped working again. Great find!

**Comment by wilkerlucio** [comment]
I have seen some people having this issue in the past so I updated the README to include the instructions to unblock third part cookies.

**Comment by avocade** [comment]
Great, thanks @wilkerlucio
